tests/integration/security/jwt_test.go

```diff
@@ -325,3 +326,130 @@ func TestAuthnJwt(t *testing.T) {
 			}
 		})
 }
+
+// TestRequestAuthentication tests beta authn policy for jwt.
+func TestRequestAuthentication(t *testing.T) {
+	testIssuer1Token := jwt.TokenIssuer1
+
+	framework.NewTest(t).
+		RequiresEnvironment(environment.Kube).
+		Run(func(ctx framework.TestContext) {
+			ns := namespace.NewOrFail(t, ctx, namespace.Config{
+				Prefix: "req-authn",
+				Inject: true,
+			})
+
+			// Apply the policy.
+			namespaceTmpl := map[string]string{
+				"Namespace": ns.Name(),
+			}
+			jwtPolicies := tmpl.EvaluateAllOrFail(t, namespaceTmpl,
+				file.AsStringOrFail(t, "testdata/requestauthn/simple-jwt-policy.yaml.tmpl"),
```

Comment:
A brief comment for what are set by the yaml would be useful to understand the setup without scroll to the resource.

Reply:
switched to more descriptive name instead.